projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Returns a notice box for displaying notices about record count and plan limits
+ *
+ * @param {object} props - record count, record limit, hasbeenindexed, hasvaliddata, hasItems
```

Comment:
It's a good idea to break down this into the individual props - you can see an example in the main RecordMeter component:

```
 * @param {object} props - Props
 * @param {number} props.postCount - Post count
 * @param {object} props.postTypeBreakdown - Post type breakdown (post type => number of posts)
 * @param {number} props.tierMaximumRecords - Max number of records allowed in user's current tier
```